pylxd/client.py

```diff
@@ -214,9 +216,9 @@ def __init__(
                 # Extra trailing slashes cause LXD to 301
                 endpoint = endpoint.rstrip('/')
                 if cert is None and (
-                        os.path.exists(self.DEFAULT_CERTS[0]) and
-                        os.path.exists(self.DEFAULT_CERTS[1])):
-                    cert = self.DEFAULT_CERTS
+                        os.path.exists(DEFAULT_CERTS[0]) and
+                        os.path.exists(DEFAULT_CERTS[1])):
```

Comment:
I think we have an opportunity to improve the readability of the code by using a `namedtuple()`.  See [collections.namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) for details.  Basically, instead of `DEFAULT_CERTS[1]` it would be possible to write `DEFAULT_CERTS.cert` and `DEFAULT_CERTS.key`.  Ask me if you need further details.